pam/internal/gdm/conversation.go

```diff
@@ -72,7 +75,8 @@ func SendData(pamMTx pam.ModuleTransaction, d *Data) (*Data, error) {
 	}
 	if log.IsLevelEnabled(log.DebugLevel) && jsonValue != nil &&
 		gdmData != nil && gdmData.Type == DataType_pollResponse {
-		jsonValue = challengeRegex.ReplaceAll(jsonValue, []byte(`"challenge":"**************"`))
+		jsonValue = secretRegex.ReplaceAll(jsonValue, []byte(`"secret":"**************"`))
+		jsonValue = secretRegexOld.ReplaceAll(jsonValue, []byte(`"secret":"**************"`))
```

Comment:
Since you're there...

```suggestion
		jsonValue = secretRegex.ReplaceAllLiteral(jsonValue, []byte(`"secret":"**************"`))
		jsonValue = secretRegexOld.ReplaceAllLiteral(jsonValue, []byte(`"secret":"**************"`))
```